.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: go
```

Comment:
I don't think we actually run travis in ecs-init anymore do we? since it's not setup in agent repo this wont work will it?

Reply:
I should have clarified -- We'll have a feature branch to integrate the makefiles and clean up all the remaining stuff left over from init.  This PR is meant to be as much a wholesale snapshot merge of the two code bases.  

So the travis stuff will go away as part of the integration feature branch.

Reply:
about the github workflow steps:
https://github.com/aws/amazon-ecs-init/blob/master/.github/workflows/gitsecrets.yml is merged into the agent workflows but I removed the init build.yml and will integrate it into the feature branch.